Helper/PaymentMethods.php

```diff
@@ -632,7 +632,7 @@ public function compareOrderAndWebhookPaymentMethods(Order $order, Notification
         $notificationPaymentMethod = $notification->getPaymentMethod();
 
         // Returns if the payment method is wallet like wechatpayWeb, amazonpay, applepay, paywithgoogle
-        $isWalletPaymentMethod = $this->isWalletTxVariant($orderPaymentMethod);
+        $isWalletPaymentMethod = $this->isWalletTxVariant((string) $orderPaymentMethod);
```

Comment:
Hello @FrankStappers,

Thank you for your contribution!

There is a new method that we can use here to identify the wallet payment methods. This implementation would be safer than casting string from null value and will better align with the new payment methods implementation.

```suggestion
        $orderPaymentMethodInstance = $order->getPayment()->getMethodInstance();
        $isWalletPaymentMethod = $this->isWalletPaymentMethod($orderPaymentMethodInstance);
```

What would you think about this change?

Best Regards,
Can